source/MultiGrid/MultiGrid.jest.js

```diff
@@ -300,4 +300,50 @@ describe('MultiGrid', () => {
       expect(bottomRightGrid.style.backgroundColor).toEqual('red')
     })
   })
+  describe('scrollTop and scrollLeft', () => {
+    it('should adjust :scrollLeft for top-right and main grids when scrollLeft is used', () => {
+      const rendered = findDOMNode(render(getMarkup({
+        columnWidth: 50,
+        fixedColumnCount: 2,
+        scrollLeft: 850
+      })))
+      const grids = rendered.querySelectorAll('.ReactVirtualized__Grid')
+      const topRightGrid = grids[1]
+      const bottomRightGrid = grids[3]
+      expect(topRightGrid.scrollLeft).toEqual(850)
+      expect(bottomRightGrid.scrollLeft).toEqual(850)
+    })
+
+    it('should adjust :scrollTop for bottom-left and main grids when scrollTop is used', () => {
+      const rendered = findDOMNode(render(getMarkup({
+        columnWidth: 50,
+        fixedColumnCount: 2,
+        scrollTop: 500
+      })))
+      const grids = rendered.querySelectorAll('.ReactVirtualized__Grid')
+      const bottomLeftGrid = grids[2]
+      const bottomRightGrid = grids[3]
+      expect(bottomLeftGrid.scrollTop).toEqual(500)
+      expect(bottomRightGrid.scrollTop).toEqual(500)
+    })
+
+    it('should adjust :scrollTop and :scrollLeft when scrollTop and scrollLeft change', () => {
+      render(getMarkup({
+        columnWidth: 50,
+        fixedColumnCount: 2
```

Comment:
These are default values so you can remove them if you want. Either that, or copy them below. Otherwise- if they weren't default values- then the second render would also change `columnWidth` and `fixedColumnCount` too (which is probably not what you want, since it potentially gives you a false positive)